src/pybind/mgr/mgr_util.py

```diff
@@ -1,5 +1,8 @@
+import json
 import os
 
+from ceph.fs.earmarking import EarmarkException  # type: ignore
```

Comment:
Why is type ignore needed here?

Reply:
mypy was complaining about it so I had to ignore it since it does actually import